src/actions.rs

```diff
@@ -131,6 +145,9 @@ impl<'a> Action for Step<'a> {
             context.insert(*name, issues);
         }
 
+        // populate T-compiler meetings
+        context.insert("meetings_tcompiler", &tcompiler_meetings.unwrap());
```

Comment:
Do we want to `unwrap`? or maybe it's better to add a `Meetings couldn't be retrieved` text when this fails?

Reply:
or probably even better log the error and move on `.map_err(|e| eprintln!(" blah blah{:?}", e)).unwrap_or_default();`

I think it's not necessary to break the agenda generation on this.